README.md

```diff
@@ -5,6 +5,53 @@
 ![Flowchart](mri2mrs_flowchart.jpg)
 
 
+# Citation
+
+If you use our software in your research, please cite as below:
```

Comment:
FYI for future reference: https://guides.github.com/activities/citable-code/

Reply:
Good thing. Let's do it after you merge the pull request.